images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,27 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json
+###################################################################################
+
+Import-Module -Name ImageHelpers -Force
+$ErrorActionPreference = "Stop"
+
+$toolset = Get-ToolsetContent
+$requiredVsixs = $toolset.visualStudio.vsix
+if(-not $requiredVsixs) {
+    Write-Host "No requiered VSIX"
+    exit 1
+}

```

Comment:
Why do we fail build if `$toolset.visualStudio.vsix` is empty? It is expected case when some OS doesn't contain any vsix to install

Reply:
good point
changed to `exit 0`